app/src/main/java/io/gnosis/safe/ui/settings/app/passcode/CryptographyManager.kt

```diff
@@ -0,0 +1,185 @@
+package io.gnosis.safe.ui.settings.app.passcode
```

Comment:
it creates confusion having 2 separate classes for handling authentication. Now its HeimdallEncryptionManager and CryptographyManager. I think they should be combined